docs/tutorials/issue-tracker/07-operator-watcher.md

```diff
@@ -7,18 +7,36 @@ By default, the operator is run as a separate container alongside your grafana d
 The operator is a logical pattern which runs one or more controllers. The typical use-case for a controller is the `operator.InformerController`, which holds:
 * One or more informers, which subscribe to events for a particular resource kind and namespace
 * One or more watchers, which consume events for particular kinds
-In our case, the boilerplate code uses the `simple.Operator` operator type, which handles dealing with tying together informers and watchers (or reconcilers) in an `InformerController` for us in `cmd/operator/main.go`:
+
+In our case, the boilerplate code uses the opinionated `simple.App` App type, which handles dealing with tying together informers and watchers (or reconcilers) as Managed Kinds for us in `pkg/app/app.go`:
```

Comment:
This bit has not been updated since the refactor away from directly using `simple.App`. It could do with a redraft by someone more familiar IMO